doc/release-notes/8985-deprecate-rsync.md

```diff
@@ -0,0 +1,9 @@
+Support for rsync has been deprecated. Information has been removed from the guides for rsync and related software such as Data Capture Module (DCM) and Repository Storage Abstraction Layer (RSAL). You can still find this information in [older versions](https://guides.dataverse.org/en/6.2/developers/big-data-support.html#data-capture-module-dcm) of the guides.
+
+The following related database settings have been deprecated as well:
+
+- :DataCaptureModuleUrl
+- :DownloadMethods
+- :LocalDataAccessPath
+- :PublicInstall
```

Comment:
Was this `:PublicInstall` setting specific to/only used by this Rsync/DCM functionality?

Reply:
... I actually thought it was added for the early "compute" experiments, not for Rsync. (hence the overlap with swift - MOC cluster used it for storage back then). 
It also looks like the concept of "public" storage has been ported to individual stores since then, where it's used for "remote" stores (I believe) at least. 
In other words, I would double-check before removing/deprecating this one.

Reply:
Yeah, you're probably right. I probably need to but `:PublicInstall` back in the docs and revert the deprecation annotations. It's a bummer because I was hoping we could simplify the code in the future but like you said it seems to be used for stuff beyond rsync.

Reply:
You're right. It was added for early MOC experiments in 3a573338d90aea74410dd546b791755da608c5a0

Will revert.

Reply:
Fixed in a00d2700d7.